src/runtime/storageNG/direct-store.ts

```diff
@@ -214,12 +214,14 @@ export class DirectStore<T extends CRDTTypeRecord> extends ActiveStore<T> {
           }
         }
         const change: CRDTChange<T> = {changeType: ChangeType.Operations, operations: message.operations};
-        void this.processModelChange(change, null, this.version, false);
+        // to make tsetse checks happy
+        const noAwait = this.processModelChange(change, null, this.version, false);
```

Comment:
we started using `void` operator in the runtime codebase for indicating "fire and forget" semantics on promises. E.g.

`void this.processModelChange(change, null, this.version, false);`

WDYT?

Reply:
We found that as a recommendation on some thread related to floating promises lint check.

Reply:
Do have dueling linters? Google uses Tsetse, and Tsetse requires one of the following:

https://tsetse.info/must-use-promises

Reply:
Tsetse seems to be ok with using `void`, as it does not complain on many instances we have in the codebase. But it is indeed not what the tsetse recommends. I think we got using `void` from this thread:

https://github.com/Microsoft/TypeScript/issues/13376#issuecomment-273289748

I personally like `void`ing an unused promise, as it is very succinct, and `void` is not used for any other purpose, so it clearly signifies an intent.

Whatever we decide let's be consistent. If you feel strongly about not using `void` shall we maybe introduce `noAwait()` function? I think it would be nicer than assigning to unused variable.

Reply:
Tsetse (internal version) actually complains with an error, that's why I changed it.

Reply:
Oh, I didn't notice that `void` was already there. Sorry, ignore me, it's not a valid approach then.

Reply:
Changed to use noAwait() function